recipes/qt/5.x.x/conanfile.py

```diff
@@ -453,6 +447,24 @@ def source(self):
         )
         save(self, os.path.join(self.source_folder, "qt5", "qtbase", "mkspecs", "features", "uikit", "bitcode.prf"), "")
 
+    def generate(self):
+        pc = PkgConfigDeps(self)
+        pc.generate()
+        ms = VCVars(self)
+        ms.generate()
+        vbe = VirtualBuildEnv(self)
+        vbe.generate()
+        vre = VirtualRunEnv(self)
+        vre.generate()
+        env = Environment()
+        env.define("MAKEFLAGS", f"j{build_jobs(self)}")
+        env.prepend_path("PKG_CONFIG_PATH", self.generators_folder)
+        if self.settings.os == "Windows":
+            env.prepend_path("PATH", os.path.join(self.source_folder, "qt5", "gnuwin32", "bin"))
+        if self._settings_build.os == "Macos":
+            save(self, "bash_env", 'export DYLD_LIBRARY_PATH="%s"' % env.vars(self).get("DYLD_LIBRARY_PATH"))
+            env.define_path("BASH_ENV", os.path.abspath("bash_env"))
```

Comment:
I don't understand this block, where do you save these env vars in a script file and associate it with a specific scope? I would say it's no-op here.